tripleblindmarket/private-kit
# Tp bt onboard copy updates

#### Description:

This PR addresses two things:

Commit 1: Updates copy for the BT app from [Figma](https://www.figma.com/file/79OwaKqHvSeriExpYzAvVN/PATHS-%2F-Exploration-%2F-April-%26-May-2020?node-id=3208%3A140)

Commit 2: Moves the GPS/BT decision logic to a single location for copy changes. This will make it easier to track these differences in a central location. There are still a few places of conditional component rendering at this time.

#### Linked Cards:

Addresses Jira Card: [SAF-507](https://pathcheck.atlassian.net/secure/RapidBoard.jspa?rapidView=28&projectKey=SAF&modal=detail&selectedIssue=SAF-507)

#### How to test:

Click through the GPS and BT app to make sure there are no copy or UI regressions.
